spec/services/api/participant_declarations/index_spec.rb

```diff
@@ -96,30 +96,5 @@
         expect(subject.scope).to include(new_provider_ineligible_declaration)
       end
     end
-
-    context "when NPQ application exists for same lead provider" do
```

Comment:
same here around avoiding building the declaration using the service but actually build it to keep this test until we delete the data? happy to discuss